vars/managerPipeline.groovy

```diff
@@ -147,7 +147,6 @@ def call(Map pipelineParams) {
         options {
             timestamps()
             disableConcurrentBuilds()
-            timeout(pipelineParams.timeout)
```

Comment:
if you are removing this, you need to make sure all the stages has timeouts, otherwise some stages can get your jobs stuck

Reply:
@ShlomiBalalis , in the `longevityPipeline.groovy` you will find this definition:
```
def (testDuration, testRunTimeout, runnerTimeout, collectLogsTimeout, resourceCleanupTimeout) = [0,0,0,0,0]
```

and then, for each stage, the relevant timeout is supposed to be assigned, like:
```
timeout(time: testRunTimeout, unit: 'MINUTES') {
```
or:
```
timeout(time: collectLogsTimeout, unit: 'MINUTES') {
```

Reply:
Yeah, they do have timeouts (besides `Running Downstream Jobs`, which I don't feel like it needs a timeout)

Reply:
`Running Downstream Jobs` waits for the other jobs to finish ? or it's just a trigger ?

if it's just a trigger, I would still recommend putting a timeout, I think 5min would be suffice.

Reply:
seems like `'Clean SCT Runners` doesn't have timeout (across all of the pipelines)

also @ShlomiBalalis are you still using the artifacts on manager jobs ? maybe it's about time we'll remove it, and stop filling up jenkins master with those logs ? (now that we have all available via Argus to download ?)

```groovy
                    try {

                        dir('scylla-cluster-tests') {
                            sh "rm -rf logs"
                            copyLogsFromSctRunner('logs')
                            archiveArtifacts artifacts: 'logs/**'
                            sh "rm -rf logs"
                        }
                    } catch (Exception err) {
                        echo err.getMessage()
                        echo "Error detected during archiveArtifacts, but we will continue."
                    }
```

Reply:
> `Running Downstream Jobs` waits for the other jobs to finish ? or it's just a trigger ?
> 
> if it's just a trigger, I would still recommend putting a timeout, I think 5min would be suffice.

It's only a trigger. Added.

> seems like `'Clean SCT Runners` doesn't have timeout (across all of the pipelines)

that's true

> also @ShlomiBalalis are you still using the artifacts on manager jobs ? maybe it's about time we'll remove it, and stop filling up jenkins master with those logs ? (now that we have all available via Argus to download ?)
> 
> ```groovy
>                     try {
> 
>                         dir('scylla-cluster-tests') {
>                             sh "rm -rf logs"
>                             copyLogsFromSctRunner('logs')
>                             archiveArtifacts artifacts: 'logs/**'
>                             sh "rm -rf logs"
>                         }
>                     } catch (Exception err) {
>                         echo err.getMessage()
>                         echo "Error detected during archiveArtifacts, but we will continue."
>                     }
> ```

Removed